en/material-system/overview.md

```diff
@@ -94,40 +98,44 @@ mat.initialize({
   defines: { USE_RGBE_CUBEMAP: true }
 });
 ```
-With all these information, the `Material` is properly initialized, and ready to use in any Renderable Component.
+With this information, the `Material` is properly initialized, and ready to use in any Renderable Component.
 
-Note，**if the material is intended to be used on any kind of `SkinningModel`, be sure to enable the `USE_SKINNING` shader macro.**
+Note: **if the material is intended to be used on any kind of `SkinningModel`, be sure to enable the `USE_SKINNING` shader macro.**
 
 Knowing which `EffectAsset` is currently using, we can specify all the shader properties:
 ```ts
 mat.setProperty('cubeMap', someCubeMap);
 console.log(mat.getProperty('cubeMap') === someCubeMap); // true
 ```
-These properties are assigned inside the material, which is just an asset on itself, and have't connected to the any model.
+These properties are assigned inside the material, which is just an asset on itself, and have't connected to any model.
```

Comment:
asset on itself
-->
asset of itself

Reply:
i mean the material by itself, is just an asset, so maybe `by itself`?

Reply:
yes, __by__ works here